package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-wooden-tree",
-  "version": "1.1.0",
+  "version": "1.1.2",
```

Comment:
You might introduce some breaking changes, this is probably not a good idea. I think you should version it as `2.0.0` but maybe @Hyperkid123 or @karelhala will say `1.2.0`.

Reply:
A different data structure, it's a breaking change, so 2.0.0 would be more appropriate I think.